### PR TITLE
IBX-27 - Multimandator - User doc changes

### PR DESCRIPTION
**Pull request by dabrt**
| Question      | Answer |
| ------------- | --- |
| JIRA Ticket   | IBX-27 |
| Versions      | 4.0, master |

Changes to reflect updated functionality resulting from multiple mandator support

### CODE REVIEW

File: docs/personalization/enabling_personalization.md [hunk 1]
```diff
@@ -19,18 +20,21 @@ On the welcome screen, enter the following details:
 Select the **I have read and agree to the Terms and Conditions** checkbox, and then click **Submit**.
 Your request is sent to Ibexa, and you will receive a confirmation email in response.
 
-## Configuring customer credentials
+## Set up service parameters
 
-When you receive the confirmation email, ask your administrator to [add the credentials to your configuration](https://doc.ibexa.co/en/latest/guide/personalization/enabling_personalization/#configuring-mandator-credentials).
+When you receive the confirmation email, ask your administrator to:
```
Comment (dkorostynski) [inline docs/personalization/enabling_personalization.md:25]: Not sure "confirmation email" is the best term here and best fit. 
The customer will receive credentials so maybe we should rephrase it?

Comment (dabrt) [reply docs/personalization/enabling_personalization.md:25]: @dkorostynski 
We've been using this term before, see two lines above. We could generally call it "an email with credentials". WDYS?

**Review by dkorostynski — APPROVED** [review]

**Review by juskora — CHANGES_REQUESTED** [review]

File: docs/personalization/perso_configuration.md [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+In the Back Office, you can you can review the configuration that controls 
+the Personalization service.
+If your [user role](../site_organization/organizing_the_site.md#permissions) has 
+the `Personalization/Edit` permission that includes your website, you can modify 
+the settings according to your requirements.
 To do this, navigate to one of the pages mentioned below and edit the item that you want to modify.
 
+!!! note "Hosting multiple websites"
+
+    If you have permissions to access several websites hosted on an [[= product_name =]]
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/personalization/perso_configuration.md:12]: In a preceding paragraph, there is a link to an article that describes user roles and permissions, so I'd leave it as-is

File: docs/personalization/previewing_scenario.md [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+If your [user role](../site_organization/organizing_the_site.md#permissions) has 
+the `Personalization/View` permission that includes your website, you can see what 
+Content items/products are recommended to the end user when specific [scenarios](scenarios.md) 
+are triggered. 
+Depending on the scenario type, you might need to provide additional information 
 to see the recommendation results.
 
+!!! note "Hosting multiple websites"
+
+    If you have permissions to access several websites hosted on an [[= product_name =]]
```
[the remark this answers is not in the document]
Comment (dabrt) [reply docs/personalization/previewing_scenario.md:12]: Ditto

**Review by juskora — APPROVED** [review]

**Pull request merged**